### ISSUE TITLE
Loging every action

### ISSUE THREAD

**Issue by MirzaHasnat** [dclstn/better-snapchat#258]
Could we potentially implement a feature where every event in the Chrome extension is logged in local storage and displayed as notifications either in the content script or the extension popup?

For instance:

Notifications:

14 - 05 - 2024, 12:32 | User123 half swiped the chat
14 - 05 - 2024, 13:12 | User123 Opened the chat
14 - 05 - 2024, 15:46 | User123 typing...
... and so on.

**Comment by MirzaHasnat** [comment]
this sounds good for devs, but let me rephares my ask:

for example here you can see facebook notification bell:
![image](https://github.com/dclstn/better-snapchat/assets/40728571/982202e7-08b2-4594-a620-5236bd7fd442)

i was asking if its possible if we can show user notification history of users  opened their chats and swipe half(and other releated features)

**Comment by IsaacHatton** [comment]
oh as in somewhere in the UI?

Personally, I think putting it in this menu as a per user history would be best but do you have any other ideas?
![image](https://github.com/dclstn/better-snapchat/assets/74068072/355b2b7d-6163-4a8a-9277-8eece0ff8e03)

(the only problem is that when Snapchat updates their app, it may cause errors with the script if we try and edit existing page elements rather than just creating a floating icon like we currently do)

**Comment by dclstn** [comment]
The best solution to this is likely saving to `localStorage` and letting the user export the logs by text file. 

And as you said @IsaacHatton, manipulating the existing DOM elements is unmaintainable without consistent selectors.

**Comment by IsaacHatton** [comment]
> The best solution to this is likely saving to `localStorage` and letting the user export the logs by text file. 
> 

Yep I agree with @dclstn here, if this is done in real time I'm sure another extension could be written to look at the localStorage and that one can deal with the UI changes if that's a particually desired feature.

**Comment by IsaacHatton** [comment]
https://discord.com/channels/1217438562107002940/1217438562107002943/1248651789713014825

It has also been requested that messages content be logged
